index.js

```diff
@@ -14,6 +14,23 @@ module.exports = plugins => ({
       return config
     }, config)
   },
+// 26/11/2019 - Diogo Azevedo  
+//added support for env 
+  env(config){
+    return plugins.reduce((config, plugin) => {
+      if (plugin instanceof Array) {
+        const [_plugin, ...args] = plugin
+        plugin = _plugin(...args)
+      }
+      if (plugin instanceof Function) {
+        plugin = plugin()
+      }
+      if (plugin && plugin.env instanceof Function) {
```

Comment:
I think `env` is an object that we should mixin with all the others. At least that's what I interpreted from #3